src/main/java/net/sf/jabref/exporter/FileActions.java

```diff
@@ -527,7 +525,7 @@ public static Reader getReader(String name) throws IOException {
         if (keySet != null) {
```

Comment:
I was also a bit confused why an iterator was used. But then I'm not really good at iterators, so I didn't want to touch it to much...